algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -662,3 +687,15 @@ case class SetSizeAggregator[A](hllBits: Int, maxSetSize: Int = 10)(implicit toB
   val leftSemigroup = new HyperLogLogMonoid(hllBits)
   val rightAggregator = Aggregator.uniqueCount[A].andThenPresent { _.toLong }
 }
+
+case class SetSizeHashAggregator[A](hllBits: Int, maxSetSize: Int = 10)(implicit hash: Hash128[A])
```

Comment:
Sounds good to me. I was just lazy really (but also, SetSizeAggregator could have serialized the intermediate state with what it has on hand, using the Hash you can't, but that should be an orthogonal concern).